src/VirtoCommerce.ExperienceApiModule.DigitalCatalog/Schemas/VariationType.cs

```diff
@@ -0,0 +1,62 @@
+using GraphQL.DataLoader;
+using GraphQL.Types;
+using MediatR;
+using VirtoCommerce.CatalogModule.Core.Model;
+using VirtoCommerce.ExperienceApiModule.Core.Extensions;
+using VirtoCommerce.ExperienceApiModule.Core.Helpers;
+using VirtoCommerce.InventoryModule.Core.Services;
+using VirtoCommerce.XDigitalCatalog.Queries;
+
+namespace VirtoCommerce.XDigitalCatalog.Schemas
+{
+    public class VariationType : ObjectGraphType<Variation>
+    {
+        public VariationType(
+            IMediator mediator,
+            IDataLoaderContextAccessor dataLoader,
+            IProductInventorySearchService productInventorySearchService
+            )
+        {
+            Field(x => x.Id, nullable: false).Description("Id of variation.");
+
+            Field(x => x.Code, nullable: true).Description("SKU of variation.");
+
+            FieldAsync<AvailabilityDataType>("availabilityData", resolve: async context =>
```

Comment:
Inventory can be taken by a single query when the product load, can't it?

Reply:
Unfortunately no.